boards/arm/serpente/pinmux.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2020 Alexander Falb <fal3xx@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <drivers/pinmux.h>
+#include <soc.h>
+
+static int board_pinmux_init(struct device *dev)
```

Comment:
Could you please rebase to the latest master and make the device struct constant? See #24873

Reply:
Sure. Done